### ISSUE TITLE
Display raw markdown on wiki pages in hidden div

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1181]
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem

On the way towards client-side markdown parsing, which may be needed for wiki subsection editing (#1005), we'll need to display the raw markdown "body content" on each wiki page, so we have something to convert into HTML on the client side (currently we convert from markdown to HTML on the server side, and so the client never sees the raw markdown).

## Solution

Let's try displaying and client-side parsing markdown. First, let's display it in a hidden `<div>`:

It's currently shown here:

https://github.com/publiclab/plots2/blob/master/app/views/wiki/show.html.erb#L107

```ruby
      <div<% unless @node.has_tag('style:wide') %> style="overflow:hidden;max-width:800px;"<% end %> id="content">
      <%= raw auto_link(insert_extras(@node.latest.render_body), :sanitize => false) %>
      </div>
```

Let's make a second div like:

```ruby
      <div<% unless params[:raw] %> style="display:none;"<% end %> id="content-raw-markdown">
        <%= raw auto_link(insert_extras(@node.latest.render_body), :sanitize => false) %>
      </div>
```

The `params` part will make it possible to show the div by adding `?raw=true` to the URL of the page. 

Then we can do a follow-up issue where we use the `megamark` library: https://github.com/bevacqua/megamark

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

**Comment by pranaygupta36** [comment]
@jywarren  I'm new to open source development and want to start by working on this issue.

**Comment by jywarren** [comment]
That would be fantastic and we'd love to have your contribution!

**Comment by pranaygupta36** [comment]
@jywarren i have submitted a pull request after fixing the issue Display raw markdown on wiki pages in hidden div  #1181

**Comment by jywarren** [comment]
Great -- solved in #1187. Thank you!
